dd-sdk-android-ndk/src/main/cpp/utils/stringutils.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+#include "stringutils.h"
+
+#include <jni.h>
+#include <string>
+
+namespace stringutils {
+
+    void copy(JNIEnv *env, jstring from, char *to, size_t max_size) {
```

Comment:
You could also write:
```
void copy(JNIEnv *env, jstring from, char *to, size_t max_size) {
    const std::string to_str = copy_to_string(env, from);
    strlcpy(to, to_str.c_str(), max_size);
}
```

Reply:
Don't need this function anymore...just forgot to remove it also from the source file.